packages/plugins/users-permissions/server/services/users-permissions.js

```diff
@@ -112,7 +112,7 @@ module.exports = ({ strapi }) => ({
         }
 
         return route;
-      }).filter((route) => route.info.type === 'content-api');
+      }).filter((route) => route.info && route.info.type === 'content-api');
```

Comment:
```suggestion
      }).filter((route) => route.info?.type === 'content-api');
```

But I'm also questioning if shouldn't be left as it was to throw an error, since it looks like it only happens on invalid routes. Is there a real-world case where this happens on a valid route?